src/constants.js

```diff
@@ -0,0 +1,5 @@
+export const ADVANCE_STANDARD = 4;
+export const MIN_RACING_COUNT = 0;
+export const MIN_NUMBER = 0;
+export const MAX_NUMBER = 9;
+export const LENGTH = 1;
```

Comment:
모듈로 분리하는게 좋네요! 다음에는 constants의 의미가 더 명확할 수 있게, object로 각 수를 관리하는 방법도 추천드려용.

```suggestion
export const ADVANCE_STANDARD = 4;
export const MIN_RACING_COUNT = 0;
export const MIN_NUMBER = 0;
export const MAX_NUMBER = 9;
export const LENGTH = 1;

export const RACING_GAME = {
	RANDOM_MIN: 0,
	RAMDOM_MAX" 9,
	ADVANCE_STANDARD: 4,
}
```

아래와 같은 방식으로 하면 모듈이 많아지고, 페이지가 많아졌을때 각 의미를 조금 더 명확하게 파악할 수 있습니다~